### ISSUE TITLE
Overfitting Problem

### ISSUE THREAD

**Issue by jinyummiao** [BingCS/AtLoc#14]
Hi, I trained the model on 7Scenes by using your provided codes, but the performances were unsatisfactory. I evaluate the model on the training set and test set, it seems that the performance on the training set is quite well while the training loss is also convergence, but the performance on the test set is not fully optimized. Do you have any idea to fix this problem? 
Data augmentations like colorjit, random crop, and random masks cannot help.
